src/Components/Inquiry/Views/InquiryConfirmation.tsx

```diff
@@ -30,6 +93,41 @@ export const InquiryConfirmation: React.FC = () => {
       <Button onClick={next} width="100%">
         Continue Browsing
       </Button>
+
+      {(!shouldHideDetailsCreateAlertCTA ||
+        checkIfArtworkIsOnLoanOrPermanentCollection(saleMessage)) && (
```

Comment:
Confirming that we [plan to remove](https://artsy.slack.com/archives/C9SATFLUU/p1685549204072359?thread_ts=1685545994.567979&cid=C9SATFLUU) these checks and all the code above that supports them?

Reply:
Yes.I will remove them in next commit.